pkgs/tools/package-management/nix/default.nix

```diff
@@ -242,12 +242,9 @@ in lib.makeExtensible (self: ({
   };
 
   nix_2_18 = common {
-    version = "2.18.1";
-    hash = "sha256-WNmifcTsN9aG1ONkv+l2BC4sHZZxtNKy0keqBHXXQ7w=";
-    patches = [
-      patch-rapidcheck-shared
-      ./patches/2_18/CVE-2024-27297.patch
-    ];
+    version = "2.18.3";
+    rev = "d13707727182d3e3d62d71be45a0aa1321ba2a91";
```

Comment:
I think the correct commit is https://github.com/NixOS/nix/commit/44fb1192185cdd03343da7faa08a1c605f773419 but it is missing a tag which we should wait on

Reply:
Hm the commit you referenced is before 2.18.2 was tagged.

Reply:
🤔 I am not sure how I ended up on that commit. The maintenance branch seems to be rather empty for the 2.18.3 release https://github.com/NixOS/nix/commits/2.18-maintenance/